lms/urls.py

```diff
@@ -103,9 +103,6 @@
     # Course API
     url(r'^api/courses/', include('course_api.urls')),
 
-    # Completion API
-    url(r'^api/completion/', include('completion.api.urls', namespace='completion_api')),
-
```

Comment:
The completion API was pulled out into a separate repo, but this was left behind.